File: InventoryApp/src/app/models/order.ts
export class Order {
  id: number;
  name: string;
  contact: string;
  itemId: number;
  quantity: number;
  price: number;
  total: number;
  static fromJsonArray(jsonArray): Order[] {
    const orders = [];

    jsonArray.forEach(element => {
      const order = new Order();
      order.id = element.id;
      order.name = element.name;
      order.contact = element.contact;
      order.itemId = element.itemId;
      order.quantity = element.quantity;
      order.price = element.price;
      order.total = element.total;
      orders.push(order);
    });

    return orders;
  }
}

File: InventoryAPI/InventoryAPI/Models/Order.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace InventoryAPI.Models
{
    public partial class Order
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string Contact { get; set; }
        public long? ItemId { get; set; }
        public long? Quantity { get; set; }
        public long? Price { get; set; }
        public string Total { get; set; }

        public virtual Item Item { get; set; }
    }
}
